package/benchmarks/bbob/_bbob.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import annotations
+
+from typing import Any
+from typing import Sequence
+
+import cocoex as ex
+import optuna
+import optunahub
+
+
+class Problem(optunahub.benchmarks.BaseProblem):
```

Comment:
What about making the class name more specific, e.g., `BBOBProblem`?

Reply:
Since this class is in the `benchmarks/bbob` package and the package only contains the bbob problems,  I feel `BBOBProblem` is redundant.